src/query/ts/values.go

```diff
@@ -64,6 +68,40 @@ func (d Datapoints) ValueAt(n int) float64 { return d[n].Value }
 // DatapointAt returns the value at the nth element.
 func (d Datapoints) DatapointAt(n int) Datapoint { return d[n] }
 
+// Values returns the values representation.
+func (d Datapoints) Values() []float64 {
+	values := make([]float64, len(d))
+	for i, dp := range d {
+		values[i] = dp.Value
+	}
+
+	return values
+}
+
+// Aligned returns values aligned to given bounds.
+func (d Datapoints) Aligned(bounds models.Bounds) []Datapoints {
```

Comment:
Rename to `Align`?

Reply:
@arnikola wanted it AlignToBounds. Is that better ?

Reply:
Early bird gets the worm @benraskin92  😜